images/ubuntu-github-actions-arm64--20.04/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:20.04
+
+RUN \
+   apt-get update && \
+   DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+     cmake build-essential ccache ninja-build git wget curl zip unzip mc nano time file\
```

Comment:
Could you add `rsync` here for the extended DNN testing?

I've already rebuilt image with it and pushed to the docker registry for PR testing.

Reply:
Sure. Thanks for rebuilding the image and pushing to docker registry.